enslig-forsorger-test/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/TestsøknadBuilder.kt

```diff
@@ -70,7 +69,7 @@ class TestsøknadBuilder private constructor(
         fun setPersonalia(navnOgFnr: NavnOgFnr? = null, adresse: Adresse = defaultAdresse()): Builder {
 
             this.personalia = Personalia(
-                Søknadsfelt("Fødselsnummer", Fødselsnummer(navnOgFnr?.fødselsnummer ?: FnrGenerator.generer())),
+                Søknadsfelt("Fødselsnummer", Fødselsnummer(navnOgFnr?.fødselsnummer ?: "17898797953")),
```

Comment:
Tror det en gang blev tatt i bruk generatorn her for å unngå at vi har et explisitt fnr i koden, samtidig så er ju dette et syntetiskt så burde vel gå fint.
Kan vi trekke ut dette til en const et sted?

Reply:
La det i Testsøknad